superset-frontend/src/SqlLab/main.less

```diff
@@ -378,7 +378,9 @@ div.tablePopover {
   //double class is better than !important
   border: 1px solid @gray-light;
   font-feature-settings: @font-feature-settings;
-  font-family: @font-family-monospace;
+  // Fire Code causes problem with Ace under Firefox
+  font-family: 'Menlo', 'Courier New', 'Ubuntu Mono', 'Consolas',
+    'source-code-pro', monospace;
```

Comment:
Two small things..We could target just the bold typeface by using the selector `.ace_editor.ace_editor .ace_keyword` if we wanted to just change that typeface. Also talking with @Steejay can we try 'Lucida Console' instead of 'Courier New'? Here's the comparison for reference:
<img width="1145" alt="_DEV__Superset" src="https://user-images.githubusercontent.com/5186919/104489541-98a61f80-5584-11eb-8c4a-1e5ed143b333.png">
<img width="1098" alt="_DEV__Superset" src="https://user-images.githubusercontent.com/5186919/104489561-a196f100-5584-11eb-94f5-977148940771.png">